spirv_cross_c.h

```diff
@@ -226,6 +226,7 @@ typedef enum spvc_resource_type
 	SPVC_RESOURCE_TYPE_ACCELERATION_STRUCTURE = 12,
 	SPVC_RESOURCE_TYPE_RAY_QUERY = 13,
 	SPVC_RESOURCE_TYPE_SHADER_RECORD_BUFFER = 14,
+	SPVC_RESOURCE_TYPE_GL_PLAIN_UNIFORM = 15,
```

Comment:
You need to bump SPVC_C_API_VERSION_MINOR, as well as spirv-cross-abi-minor in CMakeLists.txt when you add anything backwards compatible to the headers.